charts/agent-control/templates/install.yaml

```diff
@@ -61,5 +62,6 @@ spec:
             {{- if .Values.agentControlDeployment.repositoryCertificateSecretReferenceName }}
                 --repository-certificate-secret-reference-name={{ .Values.agentControlDeployment.repositoryCertificateSecretReferenceName }} \
             {{- end }}
-                --chart-name={{ .Values.agentControlDeployment.chartName }}
+                --chart-name={{ .Values.agentControlDeployment.chartName }} \
+                --release-name=agent-control-deployment
```

Comment:
Release name is mandatory in the agent control cli. Hence, we have to hardcode the deployment release name here until we work on https://new-relic.atlassian.net/browse/NR-441446.

Or we could leave it hardcoded here. No problem.

Reply:
I'm not sure if there something else to be done in that task, but isn't it easier to simple include:

```yaml
agentControlDeployment:
  releaseName: "agent-control-deployment"
```

In values.yaml

Reply:
I don't understand. I meant adding it to values *also use it here** `{{ .Values.agentControlDeployment.releaseName }}` 🤔 

Maybe you don't want to do that because it isn't fully supported in Agent Control. Is that the case?